<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Invesigación de operaciones</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1183</width>
      <height>652</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 49, 61);</string>
    </property>
    <widget class="QWidget" name="pagePerl">
     <widget class="QTableWidget" name="tabla">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>361</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border:none;
text-align: center;</string>
      </property>
      <column>
       <property name="text">
        <string>Actividad</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Detalle</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Depende de</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>To</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tn</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tp</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tablaView">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>140</y>
        <width>781</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: none;
color: rgb(0, 0, 0);</string>
      </property>
      <column>
       <property name="text">
        <string>D i,j</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>O i,j</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ti0</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ti1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tj0</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tj1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MT i,j</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ml i,j</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fecha inicio tempano</string>
       </property>
       <property name="textAlignment">
        <set>AlignJustify|AlignTop</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fecha inicio tardio</string>
       </property>
       <property name="textAlignment">
        <set>AlignJustify|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>fecha final temprano</string>
       </property>
       <property name="textAlignment">
        <set>AlignJustify|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>fecha final tardio</string>
       </property>
       <property name="textAlignment">
        <set>AlignJustify|AlignVCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QSpinBox" name="canVariable">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Número de Actividades: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGenera">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(46, 137, 191);</string>
      </property>
      <property name="text">
       <string>Generar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNuevo">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>70</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(46, 137, 191);</string>
      </property>
      <property name="text">
       <string>Nuevo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalcular">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>70</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
selection-background-color: rgb(255, 129, 120);
alternate-background-color: rgb(203, 209, 185);
background-color: rgb(46, 137, 191);</string>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Método Pert</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>580</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Autor: Parraga Pazmiño Yorly Alexander</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageSimplex">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>580</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Autor: Parraga Pazmiño Yorly Alexander</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGenerar">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>110</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(70, 151, 231);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Generar</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tablaRestri">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>110</y>
        <width>441</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Cantidad de Variable:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tablaVaria">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>461</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(0, 0, 0);</string>
      </property>
      <row>
       <property name="text">
        <string>&gt;</string>
       </property>
      </row>
     </widget>
     <widget class="QPushButton" name="btnNuevo_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>480</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(70, 151, 231);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Nuevo</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="canVariable_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tablaIgual">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>110</y>
        <width>111</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnResultado">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>480</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(70, 151, 231);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Mostrar Resultado</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="canRestri">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Restricciones</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Cantidad de Restricciones: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Función Objetivo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Método Simplex</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMETODOS">
    <property name="title">
     <string>METODOS</string>
    </property>
    <addaction name="actionSimplex"/>
    <addaction name="separator"/>
    <addaction name="actionPerl"/>
   </widget>
   <addaction name="menuMETODOS"/>
  </widget>
  <action name="actionSimplex">
   <property name="text">
    <string>Simplex</string>
   </property>
  </action>
  <action name="actionPerl">
   <property name="text">
    <string>Perl</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
